### PR TITLE
Get social profiles of top students

### PR DESCRIPTION
**Pull request by KirillTim**
New example of using Stepic API.
This small example will show how to get additional information about students.
It can be easily modified to get social networks accounts for any users.

### CODE REVIEW

**Comment by vyahhi** [comment]
👍

File: examples/users_progress.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+CLIENT_SECRET = "..."
+API_HOST = 'https://stepic.org'
+
+# 2. Get a token
+AUTH = requests.auth.HTTPBasicAuth(CLIENT_ID, CLIENT_SECRET)
+RESP = requests.post('https://stepic.org/oauth2/token/',
+                     data={'grant_type': 'client_credentials'},
+                     auth=AUTH
+                     )
+TOKEN = json.loads(RESP.text)['access_token']
```
Comment (mehanig) [inline examples/users_progress.py]: у requests есть метод .json() , можно использовать его вместо json.loads() , пример тут - https://github.com/StepicOrg/Stepic-API/commit/ef7efe26f0fa3952998c199c6e121a900df312d0

**Comment by mehanig** [comment]
Looks good!
Please fix json part and I will merge it.

**Comment by mehanig** [comment]
Looks good! Thank you. 
Merged

**Pull request merged**